### PR TITLE
Issue 402 - Add checkbox list to docs

### PR DESCRIPTION
**Pull request by michellemhey**
#### Screens

RAILS:
Screen Shot 2020-07-31 at 11 08 09 AM
Screen Shot 2020-07-31 at 12 43 42 PM

Screen Shot 2020-07-31 at 11 09 12 AM
Screen Shot 2020-07-31 at 12 44 11 PM

REACT:
Screen Shot 2020-07-31 at 11 10 13 AM
Screen Shot 2020-07-31 at 12 44 54 PM

Screen Shot 2020-07-31 at 11 11 08 AM
Screen Shot 2020-07-31 at 12 45 28 PM

#### Breaking Changes

_Please reflect if your changes may break in some way (changes like removing/renaming props are examples of breaking changes)_

#### Runway Ticket URL
[https://github.com/powerhome/playbook/issues/402](url)

#### How to test this
1. Go to Playbook
2. Click on "List" in the sidebar navigation
3. Scroll down to the "Checkbox List" card
4. See a list of check boxes
5. Click the entire row for each checkbox to see if it gets checked or unchecked
6. Test these steps for the dark mode and for the light and dark mode for the React code

#### Checklist:

- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [ ] **SCREENSHOT** Please add a screen shot or two
- [ ] **SPECS** Please cover your changes with specs
- [ ] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

**Comment by kellyeryan** [comment]
Hi Michelle! I think you need to add to CHANGELOG.md. Otherwise it looks good to me!

File: lib/playbook/version.rb [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 module Playbook
   VERSION = "6.1.0"
-end
+end
```
Comment (thestephenmarshall) [inline lib/playbook/version.rb:5]:
```suggestion
end

```

File: app/pb_kits/playbook/pb_list/_list.jsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ type ListProps = {
   tabIndex?: String,
   size?: String,
   xpadding: Boolean,
+  checkbox_list: Boolean,
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_list/_list.jsx:22]: For any JSX files, you will want to use camelCase `checkboxList` like so. 

```suggestion
  checkboxList: Boolean,
```

**Comment by michellemhey** [comment]
@kellyeryan @thestephenmarshall I updated the PR with your change requests

**Comment by thestephenmarshall** [comment]
@michellemhey It appears as though the build failed [due to linting errors](https://ci.powerhrg.com/blue/organizations/jenkins/powerhome%2Fplaybook/detail/PR-941/7/pipeline#step-80-log-44). You can either manually fix those or run `docker-compose run web yarn lint --fix` to auto-resolve them.

**Comment by michellemhey** [comment]
@thestephenmarshall I'll get this fix in later today. Thanks!

**Comment by jasoncypret** [comment]
Closing in leu of #1021

**Pull request closed without merging**